cmd/wfx/cmd/config/flags.go

```diff
@@ -64,6 +65,8 @@ const (
 const (
 	preferedStorage   = "sqlite"
 	sqliteDefaultOpts = "file:wfx.db?_fk=1&_journal=WAL"
+
+	DefaultPingIntervalSSE = 15 * time.Second
```

Comment:
Hm, where does `15` come from? Educated guess? :)

Reply:
Not really a guess: nginx's default read timeout is 60 :) I've increased it to 30 and added a comment.